src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -95,9 +94,9 @@ public void testSanitizeForSaving() {
         AccountAttributes expectedAccount = createAccountAttributesToSanitize();
         actualAccount.sanitizeForSaving();
 
-        assertEquals(SanitizationHelper.sanitizeForHtml(expectedAccount.googleId), actualAccount.googleId);
```

Comment:
So your tests should also reflect the changes?

e.g. Having some values that need to be sanitised and test whether the `actualAccount.sanitizeForSaving();` work or not?